microsoft/testsuites/acc/bvt.py

```diff
@@ -0,0 +1,191 @@
+                "https://packages.microsoft.com/keys/microsoft.asc",
+            ],
+        )
+
+        # Verify if Intel DCAP client installed.
+        #
+        # <get dmesg dmesg | grep -i sgx>
+
+        dmesg = node.tools[Dmesg]
+        if "sgx" not in dmesg.get_output():
+            node.os.install_packages("dkms")
+            wget = node.tools[Wget]
+            driver_path = wget.get(
+                url="https://download.01.org/intel-sgx/sgx-dcap/1.7/"
+                f"linux/distro/ubuntu{os_version}-server/sgx_linux_x64_driver_1.35.bin",
+                filename="sgx_linux_x64_driver.bin",
+                executable=True,
+            )
+            node.execute(driver_path, sudo=True)
+
+        # Install other packages
+        # < sudo apt -y install clang-10 libssl-dev gdb libsgx-enclave-common
+        #   libsgx-quote-ex libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev \
+        #   az-dcap-client open-enclave >
+
+        if os_version == "18.04":
+            libprotobuf = "libprotobuf10"
+        else:
+            libprotobuf = "libprotobuf17"
+
+        node.os.install_packages(
+            f"clang-10 libssl-dev gdb libsgx-enclave-common libsgx-quote-ex "
+            f"{libprotobuf} libsgx-dcap-ql libsgx-dcap-ql-dev "
+            "az-dcap-client open-enclave"
+        )
+        # REMOVE install_cmd = "apt -y install clang-10 libssl-dev gdb
+        # libsgx-enclave-common libsgx-quote-ex libprotobuf10
+        # libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client
+        #  open-enclave "
+        # stdout = node.execute(install_cmd, sudo=True)
+
+        # Copy Samples from the Binary
+        #
+        # <cp -r /opt/openenclave/share/openenclave/samples ~/mysamples>
+
+        samples_folder = node.get_working_path() / "mysamples"
+        copy_cmd = "cp -r /opt/openenclave/share/openenclave/samples " + str(
+            samples_folder
+        )
+        node.execute(copy_cmd, shell=True)
+
+        # Run Hello World and Remote Attestation
+        helloworld_dir = samples_folder / "helloworld"
+        attestation_dir = samples_folder / "attestation"
+        source_command = ". /opt/openenclave/share/openenclave/openenclaverc"
+        fail_msg = "HELLO WORLD TEST FAILED"
+
+        node.tools[Make]
+        result = node.execute(
+            f"{source_command} && make build && make run",
+            cwd=helloworld_dir,
+            shell=True,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=fail_msg,
+        )
+
+        assert_that(result.stdout).described_as("error message").contains(
+            "Enclave called into host to print: Hello World!"
+        )
+
+        fail_msg = "REMOTE ATTESTATION HAS FAILED"
+        username = node.tools[Whoami].get_username()
+        node.execute(f"usermod -a -G sgx_prv {username}", shell=True, sudo=True)
+        node.execute("export SGX_AESM_ADDR=1", shell=True, sudo=True)
+        result = node.execute(
```

Comment:
Can you please add a new tool for `sgx` and move the installation there? This will make it easy to share in other test cases. Please feel free to ping me if you need help writing a tool in Lisa.